faces.py

```diff
@@ -111,11 +111,27 @@ def get_random_person_from_all_batches(cursor, current_user):
                              last_name,
                              image_url
                       FROM people
-                      WHERE person_id != %s
+                      WHERE person_id != %s AND NOT image_url LIKE '%%no_photo_%%'
                       ORDER BY random()
-                      LIMIT 4""",
+                      LIMIT 1""",
                    [current_user])
-    return [x for x in cursor.fetchall()]
+
+    person = cursor.fetchall()
+
+    cursor.execute("""SELECT person_id,
+                             first_name,
+                             middle_name,
+                             last_name,
+                             image_url
+                      FROM people
+                      WHERE person_id != %s AND person_id != %s
+                      ORDER BY random()
+                      LIMIT 3""",
+                   [current_user, person[0][0]])
+
+    hints = cursor.fetchall()
+
+    return person + hints
```

Comment:
Does... `fetchall` just return a `list` now? That's so much better! This is much clearer than the silly `[x for x in cursor.fetchall()]`.